### DISCUSSION TITLE
J4 support for phpmail withing DKIM

### DISCUSSION THREAD

**Discussion by nmousedk** [joomla/joomla-cms#32418] [2. Cold Features]
Security issues should **NOT** be reported on this repository.

If you believe you have found a security issue, please contact the Joomla Security Strike Team via email at security@joomla.org or through the contact form at https://developer.joomla.org/security/contact-the-team.html.

Please see https://developer.joomla.org/security.html for more information on how the Joomla project responds to security issues.

**Comment by nmousedk** [comment]
As subject.
It coud be nice if joomla, if using phpmail og maybe sendmail, to support dkim.
The easy is, to have that Joomla supports dkim.
Regards

**Comment by ReLater** [comment]
@nmousedk 
Have a look on 
https://github.com/joomla/joomla-cms/issues/39262#issue-1456975909
https://github.com/joomla/joomla-cms/issues/39262#issuecomment-1322213397
